blackbox/docs/appendices/release-notes/3.0.0.rst

```diff
@@ -6,6 +6,22 @@ Version 3.0.0
 
 Released on 2018/05/16.
 
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB 2.0.4 or higher
```

Comment:
@seut Do you have any input on this minimum version? I looked through the release notes for the 2.X line, but couldn't find any thing that would indicate that you couldn't go from 2.0.4 to 3.0.0. I've also run the crate-qa tests (specifically the `bwc/test_upgrade` tests) with the following upgrade path:

```
UPGRADE_PATHS = (
    (
        VersionDef('2.0.x', False),
        VersionDef('3.0.x', False),
    ),
)
```

These tests also passed.